pkg/cmd/scan.go

```diff
@@ -142,7 +142,7 @@ func NewScanCmd(opts *pkg.ScanOptions) *cobra.Command {
 	)
 	fl.StringSliceP(
 		"gcp-scope",
-		"s",
+		"",
```

Comment:
Please use `fl.StringSlice()` instead so you don't need this empty string

Reply:
Ah, yeah, makes sense. Done